packages/flutter_tools/lib/runner.dart

```diff
@@ -152,6 +156,7 @@ Future<int> _handleToolError(
   bool reportCrashes,
   String Function() getFlutterVersion,
   ShutdownHooks shutdownHooks, {
+  required bool usingLocalEngine,
```

Comment:
Instead of introducing a new instance variable, can't we just set `reportCrashes` appropriately when we have a local engine?  (ie solve this problem one level higher?)